.pre-commit-config-zh-cn.yaml

```diff
@@ -1,4 +1,3 @@
-exclude: ^tests/data/
```

Comment:
I try to keep the "pre-commit-config-zh-cn.yaml " the same with "pre-commit-config.yaml", 
for the reason
1.  "  - repo: https://gitee.com/openmmlab/mirrors-pyupgrade " can change the code. guess the version "mirrors-pyupgrade" is for python3.6, I use it in python3.8. 
2. it will change "self().__init__" in the code

Reply:
Is it needed to split it to two PR?

Reply:
We can leave it unmofieid first, because we have not determined yet.